WordPressAuthenticator/Unified Auth/View Related/SiteAddressViewController.swift

```diff
@@ -123,31 +123,57 @@ private extension SiteAddressViewController {
     ///
     func loadRows() {
         rows = [.instructions, .siteAddress]
+        let displayHintButtons = WordPressAuthenticator.shared.configuration.displayHintButtons
+        if displayHintButtons {
```

Comment:
Couldn't this just be `if WordPressAuthenticator.shared.configuration.displayHintButtons {`?